crates/build/src/util/mod.rs

```diff
@@ -101,9 +75,6 @@ where
         cmd.current_dir(path);
     }
 
-    // Allow nightly features on a stable toolchain
-    cmd.env("RUSTC_BOOTSTRAP", "1");
```

Comment:
This is only required when doing `cargo build` so have moved it to `exec_cargo_for_wasm_target`.